### ISSUE TITLE
web support

### ISSUE THREAD

**Issue by felixwortmann** [sidlatau/flutter_email_sender#62]
It would be great, if this plugin supported flutter web

**Comment by JasonChiu-dev** [comment]
This feature is really needed. Is it possible to support web?

**Comment by sidlatau** [comment]
Yes, I think it is possible. But I personally do not use flutter web yet and do not have the bandwidth to add it. Pull requests from the community are welcome!

**Comment by JasonChiu-dev** [comment]
Thank you for your reply and contribution, hope this package will support more platforms soon.

**Comment by sidlatau** [comment]
Web support was added in v10: https://github.com/sidlatau/flutter_email_sender/pull/127
